torch/csrc/jit/attributes.h

```diff
@@ -49,7 +49,7 @@ struct ScalarAttributeValue : public AttributeValue {
 
 template<typename T, AttributeKind Kind>
 struct VectorAttributeValue : public AttributeValue {
-  using ConstructorType = const std::vector<T> &&;
+  using ConstructorType = std::vector<T>;
```

Comment:
What was the motivation for changing these?

Reply:
Taking r-value references is very annoying because you can't do things like 
```cpp
std::vector<int64_t> x = ...
node1->is(attr::sizes, x);
node2->is(attr::sizes, std::move(x));
```
because it forbids making a copy in the first line. rvalues will still bind to this overload correctly, and `std::forward` should pass the value without copying I think.